tests/e2e/testsLibrary/CodeExecutionTests.ts

```diff
@@ -16,14 +16,22 @@ const topMenu: TopMenu = e2eContainer.get(CLASSES.TopMenu);
 const ide: Ide = e2eContainer.get(CLASSES.Ide);
 
 export function runTask(taskName: string, timeout: number) {
-    test( `Run command '${taskName}'`, async () => {
+    test(`Run command '${taskName}'`, async () => {
         await topMenu.runTask(taskName);
         await ide.waitNotification('has exited with code 0.', timeout);
     });
 }
 
+export function runTaskWithDialogShell(taskName: string, timeout: number, openLink: boolean, closeDialog: boolean) {
+    test(`Run command '${taskName}' expecting dialog shell`, async () => {
+        await topMenu.runTask(taskName);
+        await ide.waitDialogShell(timeout, openLink, closeDialog);
+    });
+}
+
 export function closeTerminal(taskName: string) {
     test('Close the terminal tasks', async () => {
         await terminal.closeTerminalTab(taskName);
+        await ide.closeAllNotifications();
```

Comment:
Shouldn't this method be executed before terminal.closeTerminalTab?

Reply:
Switched.